src/MIDebugEngine/AD7.Impl/AD7StackFrame.cs

```diff
@@ -455,7 +455,8 @@ int IDebugStackFrame2.GetCodeContext(out IDebugCodeContext2 memoryAddress)
         // pointer associated with the stack frame. The debugger calls EnumProperties to obtain these values in the sample.
         int IDebugStackFrame2.GetDebugProperty(out IDebugProperty2 property)
         {
-            throw new NotImplementedException();
+            property = null;
+            return Constants.E_NOTIMPL;
```

Comment:
Glass test will try to call this, if fail it Glass turns to IDebugStackFrame2.EnumProperties